### ISSUE TITLE
Remove splash screen

### ISSUE THREAD

**Issue by rolandgeider** [wger-project/flutter#162]
Seems we forgot some steps

```
W/FlutterActivityAndFragmentDelegate(23535): A splash screen was provided to Flutter, but this is deprecated. See flutter.dev/go/android-splash-migration for migration steps.
```

**Comment by Tolu007** [comment]
Hello, I would love to fix this, can you assign it to me? Thank you

**Comment by rolandgeider** [comment]
Hi! It's yours

**Comment by rolandgeider** [comment]
Closed by #199
